Source/Core/VideoBackends/OGL/PostProcessing.cpp

```diff
@@ -37,12 +37,18 @@ OpenGLPostProcessing::~OpenGLPostProcessing()
 }
 
 void OpenGLPostProcessing::BlitFromTexture(TargetRectangle src, TargetRectangle dst,
-                                           int src_texture, int src_width, int src_height,
-                                           int layer)
+                                           CropRectangle crop_rc, int src_texture, int src_width,
+                                           int src_height, int layer)
 {
   ApplyShader();
 
+  GLboolean old_scissor_enabled = glIsEnabled(GL_SCISSOR_TEST);
```

Comment:
Avoid glGet* and glIs*. They will case a thread synchronization in the driver. We usually just reset the state based on the emulation state. Here, the caller (SwapImpl in Render.cpp) never sets a scissor, so just disable it afterwards again, or do so in https://github.com/ligfx/dolphin/blob/6efb65516ed34d86b4749ca2680d24e18f205fc3/Source/Core/VideoBackends/OGL/Render.cpp#L1419